### ISSUE TITLE
[Accessibility-Critical] Users > Add user > Join Groups > listed group(s) is/are missing label/s

### ISSUE THREAD

**Issue by agagancarczyk** [keycloak/keycloak-admin-ui#2838]
### Describe the bug

"Form elements must have labels" critical issue detected for the Users > Add user > Join Groups > listed group(s) 

![Screenshot 2022-06-20 at 16 44 28](https://user-images.githubusercontent.com/4890675/174638486-4e14ea2d-86c5-4951-9748-29ace11b70e7.png)

### Version

17.0.0

### Expected behavior

aria-labelledby="display-order" label is added.

### Actual behavior

No label present.

### How to Reproduce?

Steps to reproduce:

1. Go to 'Groups' and create a new group (if none is available)
2. Go to 'Users', select "Add user" and click "Join Groups".
3. Observe that the item/s listed in the table is/are missing the ay11 label. 

### Anything else?

Testing/ Verification:

Install AXE Dev Tools Chrome extension.
Open AXE Dev Tools on the page you want to scan. Sing up for the PRO 3-day trial.
Login to your AXE DevTools
Use the "Scan PART of my page" option and scan part of the page where the bug was found.
Once fixed. Scan that part of the page again using the "Scan PART of my page" option to verify the fix.
Add a screenshot to this ticket where AXE Scan no longer shows the error.

**Comment by agagancarczyk** [comment]
1. Fixed
![Screenshot 2022-06-22 at 12 05 51](https://user-images.githubusercontent.com/4890675/175014580-aaf54869-16d9-49a0-bfce-96e67a1ee826.png)

2. Fixed
![Screenshot 2022-06-22 at 12 07 59](https://user-images.githubusercontent.com/4890675/175014808-412f670e-c42e-4197-9755-ee057a174c15.png)

3. Fixed
![Screenshot 2022-06-22 at 12 09 33](https://user-images.githubusercontent.com/4890675/175014978-b68f96fa-38f8-4fa2-ada1-8bf63526b8ec.png)

4. Fixed
![Screenshot 2022-06-22 at 12 05 51](https://user-images.githubusercontent.com/4890675/175014580-aaf54869-16d9-49a0-bfce-96e67a1ee826.png)

5. Fixed
![Screenshot 2022-06-22 at 12 10 42](https://user-images.githubusercontent.com/4890675/175015158-f1dc5001-b197-40fd-b9d3-35b5b65ff4ea.png)
